docs/conf/eu/2017/news/recap-retrospective.rst

```diff
@@ -0,0 +1,90 @@
+:template: 2017/eu-content.html
+
+.. post:: Oct 16, 2017
+   :tags: 2017, prague, thank you, feedback
+
+Write the Docs Prague 2017 - Recap and Retrospective
+====================================================
+
+Hi everyone,
+
+Thanks for attending the 2017 edition of Write the Docs Prague! We're so grateful to our speakers, sponsors, volunteers, and attendees for making the conference possible.
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that gathers under our roof. Below we've included all of the resources you might need for staying in touch and continue to build and grow the community.
+
```

Comment:
Alright, this is the tiniest of nitpicks, but when I'm summarizing the contents of something, I always try to list them in the same order they appear below. So I'd start with the CoC report here, and then mention the resources for staying in touch.